package.json

```diff
@@ -38,6 +38,8 @@
     "lodash": "^4.17.21",
     "lodash-es": "^4.17.21",
     "ngx-avatar": "^4.1.0",
+    "ngx-countup": "^7.3.3",
+    "ngx-typed-js-public": "^1.2.0",
```

Comment:
There is another library (`ngx-typed-js`) that is more popular, but I could not get it to work.  I kept getting an error about using private properties when I added the `[strings]` attribute.  I found this [SO post](https://stackoverflow.com/a/43177386), but I couldn't really understand it -- perhaps it will make more sense to you?

Reply:
Let's keep this library for now and review later together what it's used for and if we can find another solution. Can you open a ticket in this repo to remind us to review the use of this library?

Reply:
Sure. It's just for the interactive typing seen in the Landing section.  There is a pure JS vanilla version I could use too, but this was easier to implement.